src/Firebase.cpp

```diff
@@ -118,15 +118,15 @@ FirebaseCall::FirebaseCall(const String& host, const String& auth,
   }
 }
 
+const JsonObject& FirebaseCall::parseResponse() {
```

Comment:
I was thinking to hide the fact it is json at all, but it is a JsonObject and at this point you need to know how the json response was structured in order to understand what you can access anyway so we can't really hide it.
Switching it back.